backend/src/main/routes/AccountRoute.ts

```diff
@@ -206,7 +252,11 @@ export default class AccountRoute extends CommonRoutesConfig {
       })
       .delete(async (req: express.Request, res: express.Response, next: express.NextFunction) => {
         try {
-          await this.clientAccountService.deleteClientAccountByEmail(req.params.email);
+          const authorizationToken: string = req.headers['authorization'] as string;
```

Comment:
In the frontend authorization has a capital A. I think case matters.

Reply:
I verified and it's not cap sensitive. Hence, it works with a lowercase "A".